### ISSUE TITLE
Collection of minor issues in the administration area

### ISSUE THREAD

**Issue by connium** [digitalfabrik/entitlementcard#2744]
### Describe the Bug

**🐛 No. 1 - User management:** When editing the own user, a warning is shown. This has a low contrast.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/c0503e3f-cf63-443c-a16f-218c02d8291d" />

**🐛 No. 2 - Project management (as external API user):** The button for creating a new token IMO should be a primary button.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/a52b7578-07f4-4f10-a84f-c0ba2b99632e" />

**🐛 No. 3 - Application management:** When deleting an application, a confirmation dialog is displayed. After confirmation, there is no notification shown (e.g. "Successfully deleted"). For other actions, we show such notifications.

**🐛 No. 4 - Card creation:** When importing cards from a CSV file, it is possible, to define a gold card with an expiry date. This looks strange. IMO this should be a validation error.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/dae5cae1-9d31-4dac-9a0d-db18f761ac9d" />

**🐛 No. 5 - Acceptance store management:** When importing acceptance stores, the messages states that longitude and latitude are required. While the columns must be present, they don't need to contain any data. So they shouldn't be mentioned as required fields.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/a2df1135-a4b5-48b8-995e-146f3faec0d7" />

**🐛 No. 6 - Acceptance store management:** When trying to import an empty file or a file with just the column titles, an error notification is shown. It feels like this notification vanishes much faster than others (approx. 2 seconds).

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/60da44d6-f7db-4263-818a-6f8e2aa985fb" />

**🐛 No. 7 - Acceptance store management:** If the file contains invalid data, the corresponding cells are highlighted, but there is no error notification. Especially if the error(s) are in a column far right, I don't get it. If the geolocation cannot be resolved, we show an error notification.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/788b2c3a-a585-46e3-853b-e6e277ab713f" />

**🐛 No. 8 - Acceptance store management:** When executing a dry run or actually importing stores, a notification is shown. Shouldn't this notification be a green one (success)? @hauf-toni 

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/cb569206-a0f8-48bd-9507-b995fb661630" />

**Comment by f1sh1918** [comment]
Some open questions @connium 

#### No 1. 
Is a general issue
These are the default mui alert outlined.
https://mui.com/material-ui/react-alert/#variants
If we don't want to use this, we need a design decision and adjust all of them IMO
@ulyanahoy-cloud may have some input to that
In general i would tend to solve that in a separate issue if we want to tackle alerts in general

#### No2.
Is already fixed in current version

#### No7.
I think we had the discussion to replace this table quite soon. So i think we might skip implementing a proper validation at the moment?

**Comment by connium** [comment]
> **No 1.**
> Is a general issue
> These are the default mui alert outlined.

@f1sh1918 Can you please double-check, if we don't overwrite the defaults? I inspected the colors (border and text) on the MUI page, and they are darker compared to ours.

> **No 2.**
> Is already fixed in current version.

Nice!

> **No 7.**
> I think we had the discussion to replace this table quite soon. So i think we might skip implementing a proper validation at the moment?

Valid point. I forgot to update the issue. No need to do anything here. ✔️

**Comment by ulyanahoy-cloud** [comment]
> No 1.
Is a general issue
These are the default mui alert outlined.
https://mui.com/material-ui/react-alert/#variants
If we don't want to use this, we need a design decision and adjust all of them IMO

I think using the standard MUI alert variant with a light semantic background would work better here than the outlined variant. It gives the alert more visual separation and makes the status easier to recognize. I would use this consistently for all alert types and check the final theme colors for sufficient contrast. What do you think?

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/d088d024-0804-472d-ab2e-50f1bbf0072a" />

For “Sie bearbeiten ihr eigenes Konto”, I would also use warning instead of error. ( orange)

**Comment by connium** [comment]
@ulyanahoy-cloud should we use the MUI alert variant with a light semantic background everywhere for every type of alert as shown in your screenshot?

**Comment by f1sh1918** [comment]
Okay it looks like we are overriding the light colors in our theme (not using standard mui colors).
We could use the main color for that which would result in

<img width="596" height="450" alt="Image" src="https://github.com/user-attachments/assets/61b526c6-a49d-4267-bd9f-7b0213125faa" />

 But the root cause of this problem is that we define these colors in our color system @connium 
IMO we should think about adjusting that instead of now manually overriding it for the Alert Form
```
import { PaletteOptions } from '@mui/material'

export const commonColors: PaletteOptions = {
  mode: 'light',
  primary: {
    main: '#073693',
    light: '#99A7D9',
  },
  error: {
    main: '#dd1d1a',
    light: '#ffcbce',
  },
  warning: {
    main: '#ff9706',
    light: '#ffb64e',
  },
  success: {
    main: '#4CAF50',
    light: '#81C784',
  },
  info: {
    main: '#0096c3',
    light: '#d7edf5',
  },
  text: {
    primary: '#1c2127',
    secondary: '#5C6065',
    disabled: '#898989',
  },
}

```

So actually using the default variant would lead into this

<img width="596" height="451" alt="Image" src="https://github.com/user-attachments/assets/5933f044-3038-419f-ad70-37f768a28a08" />

Hier mal die Kontrastwerte @ulyanahoy-cloud 


  | Farbe | `.light` (aktuell) | Lightness | Sättigung | Kontrast vs. Weiß |
  |---|---|---:|---:|---:|
  | **warning** | `#ffb64e` | 65% | 100% | 1.74:1 |
  | **success** | `#81C784` | 64% | 38% | 2.01:1 |
  | **error** | `#ffcbce` | 90% | 100% | 1.43:1 |
  | **info** | `#d7edf5` | 90% | 60% | 1.21:1 |
  | primary | `#99A7D9` | 73% | 46% | 2.36:1 |

**Comment by connium** [comment]
Thanks for the research. I totally agree, that we shouldn't add overrides just for the Alert.

Let's **skip issue no. 1** for now, and I'll create a **separate ticket** to address the alert style and the error/warning/success/info colors.

**Comment by f1sh1918** [comment]
> Thanks for the research. I totally agree, that we shouldn't add overrides just for the Alert.
> 
> Let's **skip issue no. 1** for now, and I'll create a **separate ticket** to address the alert style and the error/warning/success/info colors.

Well i just talked to Sina and she said that the border is not really the important thing but the text color to the background color and there the contrast is actually 4.5-7.0 so good enough @connium 
I guess mui will lighten or darken the particular colors depending on their use so f.e. text (darken) and border(lighten) and that results in that difference

But in general it might make sense to have the same contrast values for light colors within our color system so i guess if we use 2.00 at least the borders. so all texts should meet our a11y requirements
